src/main/java/com/synopsys/integration/alert/channel/email/descriptor/EmailGlobalDescriptorActionApi.java

```diff
@@ -56,10 +57,10 @@ public EmailGlobalDescriptorActionApi(final EmailChannel emailChannel) {
 
     // TODO Global email config doesn't validate properly or give any indication that saving was successful
     @Override
-    public void validateConfig(final FieldAccessor fieldAccessor, final Map<String, String> fieldErrors) {
-        final String port = fieldAccessor.getString(EmailPropertyKeys.JAVAMAIL_PORT_KEY.getPropertyKey()).orElse(null);
-        final String connectionTimeout = fieldAccessor.getString(EmailPropertyKeys.JAVAMAIL_CONNECTION_TIMEOUT_KEY.getPropertyKey()).orElse(null);
-        final String timeout = fieldAccessor.getString(EmailPropertyKeys.JAVAMAIL_TIMEOUT_KEY.getPropertyKey()).orElse(null);
+    public void validateConfig(final FieldModel fieldModel, final Map<String, String> fieldErrors) {
+        final String port = fieldModel.getField(EmailPropertyKeys.JAVAMAIL_PORT_KEY.getPropertyKey()).flatMap(field -> field.getValue()).orElse(null);
```

Comment:
It seems we are calling `fieldModel.getField(...).flatMap(field -> field.getValue())` all over the place, should we add a method to FieldModel that does this?